packages/mask/src/plugins/Trader/SNSAdaptor/trending/TrendingViewDeck.tsx

```diff
@@ -187,6 +187,11 @@ export function TrendingViewDeck(props: TrendingViewDeckProps) {
     const [coinMenuOpen, setCoinMenuOpen] = useState(false)
     const coinAddress = coin.address || coin.contract_address
 
+    const displayResultList = uniqBy(
```

Comment:
Move this to `packages/web3-providers/src/DSearch/index.ts`.

Reply:
This is work for ui. 
If we move to `packages/web3-providers/src/DSearch/index.ts`.
Give the token A from coingecko / token A' from CMC / token B
The search result will return the token A and token B. We will lose the token A' which from CMC.

Reply:
ok